scalardb-test/src/main/java/kelpie/scalardb/transfer/TransferWithTwoPhaseCommitTransactionProcessor.java

```diff
@@ -58,7 +60,9 @@ public void executeEach() throws Exception {
 
   @Override
   public void close() {
-    manager.close();
+    manager1.close();
+    manager2.close();
```

Comment:
@komamitsu I will do what you commented in the following in another PR:
https://github.com/scalar-labs/kelpie-test/pull/74#discussion_r1090132289